src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -342,6 +378,38 @@ public function flush()
         return true;
     }
 
+    /**
+     * Remove all items from the cache.
+     *
+     * @param  string  $key
+     * @return bool
+     */
+    protected function forgetMany(array $keys)
```

Comment:
/**
     * Remove all items from the cache.
     */
    protected function forgetMany(array|string $keys): bool